lib/web/desktop.go

```diff
@@ -58,7 +59,7 @@ import (
 	"github.com/gravitational/teleport/lib/web/scripts"
 )
 
-// GET /webapi/sites/:site/desktops/:desktopName/connect?access_token=<bearer_token>&username=<username>
+// GET /webapi/sites/:site/desktops/:desktopName/connect/ws?username=<username>
```

Comment:
I realize this is not a new addition introduced in this PR, but sending `username` in URL caught my eye. This just risks exposing username (especially email addresses) in unwanted places. Why not `userid`?